lib/components/src/syntaxhighlighter/syntaxhighlighter.tsx

```diff
@@ -124,21 +124,26 @@ export const SyntaxHighlighter: FunctionComponent<Props> = ({
   className = null,
   ...rest
 }) => {
+  if (typeof children !== 'string' || !children.trim()) {
```

Comment:
I've added this check `!children.trim()` to also return `null` instead of a component for strings which contain whitespaces only.